app/Makefile

```diff
@@ -33,15 +33,37 @@ else
 PY_RUN_CMD := docker-compose run $(DOCKER_EXEC_ARGS) --rm $(APP_NAME) poetry run
 endif
 
+# Docker user configuration
```

Comment:
Do you mean change the comments in the code a bit to make it clearer + what Tanner said?

Reply:
Yeah exactly. The comment explains what it's doing (configure Docker users) but not why (we don't want Dockerfile to run as root user). I'm not sure the best place to put the comment, here or in the Dockerfile, but somewhere it would be useful to explain what Tanner described - i.e. the thing about mounted volumes needing to be owned by the same UID, and the thing about not using the root user in Dockerfile

Reply:
Slightly tweaked the comments to try and make it clearer